smart_contracts.rst

```diff
@@ -165,51 +167,38 @@ Allows a channel participant to close the channel. The channel cannot be settled
 
     Only a valid signed balance proof from the channel partner (the other channel participant) must be accepted. This balance proof sets the amount of tokens owed to the participant by the channel partner.
 
-**Update transfer state**
+**Update non-closing participant balance proof**
 
-Called after a channel has been closed. Allows the non-closing participant to provide a balance proof for the latest transfer from the closing participant. This modifies the state for the closing participant.
+Called after a channel has been closed. Allows the non-closing participant to provide the latest balance proof from the closing participant. This modifies the stored state for the closing participant.
 
 ::
 
-    function updateTransfer(
+    function updateNonClosingBalanceProof(
         uint channel_identifier,
         uint64 nonce,
-        uint256 transferred_amount,
-        bytes32 locksroot,
-        bytes32 additional_hash,
-        bytes closing_signature)
-        public
-
-    function updateTransferDelegate(
-        uint channel_identifier,
-        uint64 nonce,
-        uint256 transferred_amount,
-        bytes32 locksroot,
-        bytes32 additional_hash,
+        bytes32 balance_hash,
         bytes closing_signature,
         bytes non_closing_signature)
         public
 
 ::
 
-    event TransferUpdated(
+    event NonClosingBalanceProofUpdated(
         uint256 channel_identifier,
         address closing_participant
     );
 
 - ``channel_identifier``: Channel identifier assigned by the current contract.
 - ``nonce``: Strictly monotonic value used to order transfers.
-- ``transferred_amount``: The monotonically increasing counter of the closing participant's amount of tokens sent.
-- ``locksroot``: Root of the merkle tree of all pending lock lockhashes for the closing participant.
-- ``additional_hash``: Computed from the message. Used for message authentication.
+- ``balance_hash``: Hash of the balance data
 - ``closing_signature``: Elliptic Curve 256k1 signature of the closing participant on the balance proof data.
 - ``non_closing_signature``: Elliptic Curve 256k1 signature of the non-closing participant on the balance proof data.
```

Comment:
For these two signatures we are also using `Hash of the balance data` instead of just the `balance data`. I would also add that the `non_closing_signature` is computed on `hash(nonce || balance_hash || channel_identifier || token_network_address || chain_id || closing_signature)`